web/src/tsconfig.app.json

```diff
@@ -6,8 +6,5 @@
     "types": ["jest"]
   },
   "files": ["main.ts", "polyfills.ts"],
-  "include": ["src/**/*.d.ts"],
-  "angularCompilerOptions": {
-    "enableIvy": false
-  }
```

Comment:
what's this and why did you remove this?

Reply:
Angular switched in version 9 from their ViewEngine to Ivy as their default rendering pipeline.
We used this option to force the use of the ViewEngine, but this option has been removed as of Angular 11.
All this option currently does is produce a warning when starting our frontend.